src/toil/lib/docker.py

```diff
@@ -0,0 +1,238 @@
+import base64
+import logging
+import os
+import subprocess
+
+from bd2k.util.exceptions import require
+
+_logger = logging.getLogger(__name__)
+
+
+def dockerCall(job, tool, parameters=None, workDir=None, dockerParameters=None, outfile=None, defer=None):
+    """
+    Wrapper call to Docker. See docstring for Docker()
```

Comment:
This method throws a `CalledProcessError` if the Docker invocation has a non-zero exit code, right? Let's document that.